benchmark/opperf/utils/op_registry_utils.py

```diff
@@ -196,6 +196,30 @@ def get_all_broadcast_binary_operators():
     return binary_broadcast_mx_operators
 
 
+def get_all_misc_binary_operators():
+    """Gets all miscellaneous binary operators registered with MXNet.
+
+    Returns
+    -------
+    {"operator_name": {"has_backward", "nd_op_handle", "params"}}
+    """
+    # Get all mxnet operators
+    mx_operators = _get_all_mxnet_operators()
+
+    # Filter for miscellaneous binary operators
+    binary_misc_mx_operators = {}
+    for op_name, op_params in mx_operators.items():
+        if "choose_element_0index" == op_name and op_params["params"]["narg"] == 5 and \
```

Comment:
why do we need to check for other params?

Reply:
Good point - removing extraneous checks.